src/main/java/Validate.java

```diff
@@ -0,0 +1,46 @@
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public class Validate {
+    public void isValidCarNames(String[] carNames) {
+        checkBlank(carNames);
+        checkLength(carNames);
+        checkDouble(carNames);
+    }
+
+    public void checkDigits(String input) {
+        boolean isMatch = input.matches("\\d+");
+        if (!isMatch) {
+            throw new IllegalArgumentException("[ERROR] : 숫자만 입력이 가능합니다.");
+        }
+    }
+
+    public void checkRange(int tryCount) {
+        if (tryCount < 1) {
+            throw new IllegalArgumentException("[ERROR] : 1이상의 정수만 가능합니다.");
+        }
+    }
+
+    private void checkBlank(String[] carNames) {
+        if (carNames.length == 0) {
+            throw new IllegalArgumentException("[ERROR] : 빈 이름은 들어올 수 없습니다");
+        }
+    }
+
+    private void checkLength(String[] carNames) {
+        long invalidCount = Arrays.stream(carNames)
+                .filter(names -> names.isEmpty() || names.length() > 5)
```

Comment:
carNames에서 이름을 받았으므로 stream 과정에는 name이 되어야할 것 같아요!
```suggestion
                .filter(name -> name.isEmpty() || name.length() > 5)
```

Reply:
확인했습니다 !